specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/examples/DataRegistries_RegisterOrReplace.json

```diff
@@ -0,0 +1,88 @@
+{
+  "parameters": {
+    "geography": "us",
+    "api-version": "2022-12-01-preview",
+    "udid": "29a92cab-74af-49e7-a326-66898220aa9d",
+    "dataRegistry": {
+      "description": "Contoso Geofence GeoJSON",
+      "dataFormat": "geojson",
+      "connection": {
+        "type": "AzureBlobFile",
+        "msiClientId": "3263cad5-ed8b-4829-b72b-3d1ba556e373",
+        "linkedResource": "my-storage-account",
+        "scope": {
+          "containerName": "my-container",
+          "blobPath": "/my/blob/path.json"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "udid": "29a92cab-74af-49e7-a326-66898220aa9d",
+        "description": "Contoso Geofence GeoJSON",
+        "dataFormat": "geojson",
+        "connection": {
+          "type": "AzureBlobFile",
+          "msiClientId": "3263cad5-ed8b-4829-b72b-3d1ba556e373",
+          "linkedResource": "my-storage-account",
+          "scope": {
+            "containerName": "my-container",
+            "blobPath": "/my/blob/path.json"
+          }
+        },
+        "downloadURL": "https://us.atlas.microsoft.com/mapData/29a92cab-74af-49e7-a326-66898220aa9d",
+        "sizeInBytes": 1339,
+        "contentMD5": "sOJMJvFParkSxBsvvrPOMQ==",
```

Comment:
the contentMD5 would be null because we fill in it in background job.

Reply:
Fixed